packages/amesos2/src/Amesos2_Superlu_FunctionMap.hpp

```diff
@@ -79,6 +79,19 @@ namespace SLU {
 #include "slu_util.h"
 #include "superlu_enum_consts.h"
 
+  #ifdef HAVE_TEUCHOS_COMPLEX
```

Comment:
This is strange, I don't know why these were working before need to be added here. Can you please explain ?

Reply:
I moved the includes from Type_Map to FunctionMap since they were not needed in TypeMap anymore. However maybe the changes in TypeMap I did are not correct. We used to be able to build the solver templated over the native SLU type. But now that we are using views I didn't think we would ever want a view of that type. I'm not sure that can even work with Kokkos unless we added support in a lot of places. This is a backwards compatibility issue though. 

Would user ever want to build the solver directly on the native SLU type? They can't do that anymore.

Would we ever want to have a view of data which was the native SLU type?

If what I did for TypeMap is ok, then a lot of the other questions follow from that.